internal/types/document.go

```diff
@@ -321,39 +321,42 @@ func (d *Document) Remove(key string) {
 
 // RemoveByPath removes document by path, doing nothing if the key does not exist.
 func (d *Document) RemoveByPath(keys ...string) {
+	removeByPath(d, keys...)
+}
+
+func removeByPath(v any, keys ...string) {
```

Comment:
Should we add comment to this function?

Reply:
it's private, the public one with similar name has.